src/networking.c

```diff
@@ -40,8 +40,23 @@
 #include <sys/uio.h>
 #include <math.h>
 #include <ctype.h>
+#include <stdbool.h>
 #include <stdatomic.h>
 
+typedef enum {
+    CLIENT_REPLY_PAYLOAD_DATA = 0,
+    CLIENT_REPLY_PAYLOAD_BULK_OFFLOAD,
+} clientReplyPayloadType;
```

Comment:
```suggestion
} clientReplyType;
```
You use the term payload which seems unnecessary. The noun is the Reply.

Reply:
Applied suggestion